webapp/src/components/github_repo_selector/github_repo_selector.jsx

```diff
@@ -57,13 +50,13 @@ export default class GithubRepoSelector extends PureComponent {
                     limitOptions={true}
                     required={true}
                     onChange={this.onChange}
-                    options={this.state.repoOptions}
+                    options={repoOptions}
                     isMulti={false}
-                    key={'LT'}
+                    key={'repo'}
                     theme={this.props.theme}
                     addValidate={this.props.addValidate}
                     removeValidate={this.props.removeValidate}
-                    value={this.state.repoOptions.find((option) => option.value === this.props.value)}
+                    value={repoOptions.find((option) => option.value === repoOptions.value)}
```

Comment:
Found the issue :wink:

```suggestion
                    value={repoOptions.find((option) => option.value === this.props.value)}
```

Reply:
I knew you would 😡 :smile: